src/components/OSCALControlModification.js

```diff
@@ -42,13 +42,30 @@ function getAlterAddsOrRemovesDisplay(addsElements, addsLabel, controlPartId) {
         Name: {item.name}, Value: {item.value}
       </Typography>
     ));
+  let removedTypographies = null;
+  if (addsLabel === "Removes ") {
```

Comment:
If `addsLablel === "Removes "` is it really an **adds** label? Same for `addsElements`. 

The function documentation also describes the `addsLabel` as a boolean variable which makes this more confusing.

Reply:
This label should be string, I changed it to the more descriptive addsRemovesLabel.

Reply:
Yeah this string comes from https://github.com/EasyDynamics/oscal-react-library/blob/57b305bba845f4c6593549d57246f23248e8b8d8/src/components/OSCALControlModification.js#L145-L163 and it's an odd architectural decision...